### PR TITLE
feat: adding view profile Megloblin

### PR DESCRIPTION
**Pull request by Dbishop15**
## Describe your changes
Create a "Me Goblin" tab on the user's profile page to display their wishlist and watched-list, allowing them to view their activity and watchlist.
## Issue ticket number and link

Slop-13:
https://apiary-edtonomy.atlassian.net/jira/software/projects/SLOP/boards/2?selectedIssue=SLOP-13

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] If there are changes to components, I have added / updated automated tests
- [ ] I have made any necessary updates to Storybook to accompany these changes
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

**Review by SamPage4427 — COMMENTED** [review]
Everything looks good so far. I would just put the code into the tabber folder components since that is where they should be and change the tabber so it looks like the figma.

**Review by Dillona25 — APPROVED** [review]
Everything looks great and the ticket is handled! Though Sam has some comments that I would 2nd here!

File: src/components/tabber/tabber.jsx [hunk 1]
```diff
@@ -1,13 +1,36 @@
...
+import { useContext, useMemo } from "react";
+import { GET_USER_WATCHLIST } from "../../graphql";
+import { CurrentUserContext } from "../../store/current-user-context";
 import { TabList } from "./tab-list";
-import { TabPanel } from "./tab-panel";
 import { TabPanels } from "./tab-panels";
 
-export const Tabber = ({ children }) => {
-  return <Tab.Group>{children}</Tab.Group>;
+export const Tabber = () => {
```
Comment (jahorwitz) [inline src/components/tabber/tabber.jsx:9]: This component is used in several places and is meant to be re-usable. We should leave it like it was and then create components in the /routes directory that import the tabber components from here and makes the query for the watchlist, sets up the tab list and panels, etc. Same goes for the tab list and panels-- everything in the /components directory needs to remain re-useable or else we will break other parts of the app that are using this component

**Review by jahorwitz — CHANGES_REQUESTED** [review]
Once you refactor the tabber, drop this ticket back into final-review and I'll take another look! 👍

**Review by jahorwitz — APPROVED** [review]

**Pull request merged**